CONTRIBUTING.md

```diff
@@ -27,6 +47,67 @@ pull request reviewed, the reviewer(s) may ask you to complete additional
 work, tests, or other changes before your pull request can be ultimately
 accepted.
 
+# For maintainers
+
+## Setting up environment
+
+Prerequisites:
+
+- Dart SDK installed in the path, [standalone][dart_standalone_link], or included in the [Flutter SDK][dart_on_flutter_link].
```

Comment:
We should inform what version of the Dart SDK the user should install (ideally forwarding them to the document where this is specified to avoid redundancy), in other CONTRIBUTING files we use a text like the one below:

```suggestion
- Install a valid [Dart SDK](https://dart.dev/get-dart) in your local environment, it should be compatible with the latest version of [Dart Frog CLI](https://github.com/VeryGoodOpenSource/dart_frog/blob/main/packages/dart_frog_cli/pubspec.yaml). If you have Flutter installed, you likely have a valid Dart SDK version already installed.
```